Quickstart/azure-search-quickstart/azure-search-quickstart/Address.Methods.cs

```diff
@@ -10,17 +10,22 @@ public partial class Address
         // You can override ToString() in your own model class if you want, but you don't need to in order
         // to use the Azure Search .NET SDK.
 
-        public override string ToString()
-        {
-            var builder = new StringBuilder();
+        public override string ToString() =>
+            IsEmpty ?
+                string.Empty :
+                $"{StreetAddress}\n{City}, {StateProvince} {PostalCode}\n{Country}";
 
-            if (!IsEmpty)
-            {
-                builder.AppendFormat("{0}\n{1}, {2} {3}\n{4}", StreetAddress, City, StateProvince, PostalCode, Country);
-            }
+        //public override string ToString()
```

Comment:
No need to leave commented-out code there. If we really need to get it back, that's what git history is for.